src/frontend/apps/lti_site/components/App/AppContentLoader/index.tsx

```diff
@@ -73,17 +73,24 @@ const intl = createIntl(
   cache,
 );
 
-const appsContent: Record<string, LazyExoticComponent<ComponentType<any>>> = {};
+const ltiSite = lazy(() => import('components/LTIRoutes'));
+const appsContent: Record<string, LazyExoticComponent<ComponentType<any>>> = {
+  LTI: ltiSite,
+  Site: lazy(() => import('components/SiteRoutes')),
+};
 Object.values(appNames).forEach((app) => {
   appsContent[app] = lazy(() => import(`apps/${app}/components/Routes`));
 });
-const ltiSite = lazy(() => import('components/LTIRoutes'));
 
 const AppContentLoader = () => {
   const appConfig = useAppConfig();
   const queryClient = useMemo(() => new QueryClient(), []);
 
-  const Content = appConfig.appName ? appsContent[appConfig.appName] : ltiSite;
+  const Content = appConfig.appName
+    ? appsContent[appConfig.appName]
+    : appConfig.frontend
+    ? appsContent[appConfig.frontend]
+    : ltiSite; // fallback on LTI site
```

Comment:
combining ternary operator is really hard to read. I prefer having a good old `if/else` statement here.